CONTRIBUTING.md

```diff
@@ -227,15 +227,17 @@ To explore DevTools with all experimental features enabled:
 
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-2. Start DevTools:
+2. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet. It is ok if you already have devtools cloned somewhere else.
+
```

Comment:
why are we telling people to clone DevTools again? They should use their existing clone from the tip of master branch if they already have DevTools checked out. We should only tell a user to clone DevTools if they do not already have a local checkout

Reply:
Created other section for refresh.